### ISSUE TITLE
Unable to send messages, possibly due to upgrading server to Ubuntu 22.04

### ISSUE THREAD

**Issue by alarazr** [rahiel/telegram-send#122]
I'm getting the following result when I try to use any telegram-send command

`Traceback (most recent call last):
  File "/home/razlr/.local/bin/telegram-send", line 5, in <module>
    from telegram_send.telegram_send import main
  File "/home/razlr/.local/lib/python3.10/site-packages/telegram_send/__init__.py", line 2, in <module>
    from .telegram_send import configure, send
  File "/home/razlr/.local/lib/python3.10/site-packages/telegram_send/telegram_send.py", line 33, in <module>
    from telegram.constants import MAX_MESSAGE_LENGTH
ImportError: cannot import name 'MAX_MESSAGE_LENGTH' from 'telegram.constants' (/home/razlr/.local/lib/python3.10/site-packages/telegram/constants.py)`

I have scripts which use telegram-send, and I haven't had one since the 17th. I haven't made any changes to my server except running do-upgrade-release but I can't remember if that was on the 17th...

Any ideas what could have caused this?

**Comment by donmega** [comment]
Same error here, also on Ubuntu 22.04

**Comment by blurgyy** [comment]
@trueToastedCode's code works for sending text messages, but doesn't work for sending files/images.  (I do appreciate @trueToastedCode's efforts for making sending text messages work again!  This comment just provides a little information for those who's trying to make files sending work)

**Comment by clemens86** [comment]
Same here:
`/usr/local/lib/python3.10/dist-packages/telegram_send/telegram_send.py:246: RuntimeWarning: coroutine 'Bot.send_message' was never awaited
  message_ids += [send_message(m, parse_mode)["message_id"]]
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
Traceback (most recent call last):
  File "/usr/local/bin/telegram-send", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/telegram_send/telegram_send.py", line 125, in main
    message_ids += send(
  File "/usr/local/lib/python3.10/dist-packages/telegram_send/telegram_send.py", line 246, in send
    message_ids += [send_message(m, parse_mode)["message_id"]]
TypeError: 'coroutine' object is not subscriptable
`

**Comment by twjang** [comment]
Hi, you might have to downgrade `python-telegram-bot` package.

```
pip3 install -U python-telegram-bot==13.0
```

**Comment by Felixoid** [comment]
@tejotemku do your changes from https://github.com/rahiel/telegram-send/compare/master...tejotemku:telegram-send:master solve the mentioned here issue?

**Comment by tejotemku** [comment]
I had the same problem with coroutine on ARM processor, decided to just send telegram messages by myself. If the only thing you need this package for is sending messages you can do that via this:
```
def telegram_send_msg(message):
      url = f"https://api.telegram.org/bot{TOKEN}/sendMessage?chat_id={CHAT_ID}&text={message}&disable_web_page_preview=true"
      requests.get(url).json()
```

**Comment by alarazr** [comment]
Hi guys, I'm not sure if this will work for everyone but I followed the following steps to get it working (some may not be relevant)
```
pip3 install -U python-telegram-bot==13.0
sudo telegram-send --clean
pip3 uninstall telegram-bot
pip3 install telegram bot
```
Hopefully it works for you too

**Comment by Felixoid** [comment]
Kindly sorry, but it doesn't look to me like a solution, but rather a workaround.

The current version is completely broken with the updated python-telegram-bot.

**Comment by rahiel** [comment]
This is properly fixed in merged PR #124, a new release will follow soon where this work around is not necessary anymore.
